docs/Validation.md

```diff
@@ -61,7 +61,7 @@ Bernoulli   | 0<=p<=1   |    -      |    -
 Binomial    | 0<=p<=1   |   n>0     |    - 
 -->
 
-### `Point` Prediction Elements
+### `Point` Prediction Elements (also applies to `mean`, `median`, and `mode`)
```

Comment:
Same as comment above to feature the new types more on equal footing with point, could we make this header

### `point`, `mean`, `median`, and `mode` Prediction Elements